https.go

```diff
@@ -133,6 +137,11 @@ func (proxy *ProxyHttpServer) handleHttps(w http.ResponseWriter, r *http.Request
 		ctx.Logf("Accepting CONNECT to %s", host)
 		proxyClient.Write([]byte("HTTP/1.0 200 OK\r\n\r\n"))
 
+		if proxy.ConnectCopyHandler != nil {
+			go proxy.CopyHandler(ctx, proxyClient, targetSiteCon)
```

Comment:
```suggestion
			go proxy.ConnectCopyHandler(ctx, proxyClient, targetSiteCon)
```

Reply:
I don't know if my suggestion is correct, but since this looks like a typo, it warrants an explanatory comment if you actually meant `CopyHandler`!

Reply:
@ransford-stripe yeah, this was a typo. Nice catch!